docs/get_started/getting_started_android_cmake.md

```diff
@@ -61,15 +61,28 @@ You may also want to add the folder to the `PATH` environment variable.
 
 ## Configure and build
 
-### Configure on Linux
+### Host configuration
+
+Build and install at least the compiler tools on your host machine, or install
```

Comment:
I'd just mention building on host machine for now. We don't have binary releases; we can mention that after having it.

Reply:
We do have binary releases through the Python wheels, but they don't yet contain all the relevant compiler tools.